lib/manageiq/appliance_console/database_configuration.rb

```diff
@@ -80,7 +82,8 @@ def create_or_join_region
     end
 
     def create_region
-      ManageIQ::ApplianceConsole::Utilities.bail_if_db_connections("preventing the setup of a database region")
+      hint = "\nPlease choose \"Stop EVM Server Processes\" in appliance console."
```

Comment:
I think this should mention that this needs to be done for all appliances in the region.

Maybe "Please stop the EVM server process on all appliances in the region"?